src/scene/shader-lib/chunks/gsplat/vert/gsplatCommon.js

```diff
@@ -1,20 +1,25 @@
 export default /* glsl */`
 
-struct SplatState {
+// stores the source UV and order of the splat
+struct SplatSource {
     uint order;         // render order
     uint id;            // splat id
     ivec2 uv;           // splat uv
     vec2 cornerUV;      // corner coordinates for this vertex of the gaussian (-1, -1)..(1, 1)
 };
 
-struct ProjectedState {
+// stores the camera and clip space position of the gaussian center
+struct SplatCenter {
+    vec3 view;          // center in view space
+    vec4 proj;          // center in clip space
     mat4 modelView;     // model-view matrix
```

Comment:
With clustered lighting, I had precision issue when a mat4 was stored in a struct, so I had to move it out. Not sure it would have the same impact here, likely not as shadow flickering is a specific issue.